### DISCUSSION TITLE
Open Edition

### DISCUSSION THREAD

**Discussion by 2e0wkw** [nakhonthai/ESP32APRS_T-TWR#25] [Q&A]
I accidentally bought the Open Edition of the T-TWR will you be able to support it one day ?
I am a developer and happy to beta test.

**Comment by lyusupov** [comment]
> Open Edition of the T-TWR

ESP32APRS_T-TWR firmware version 0.3d can _(partially)_ work with SA868 'Open Edition' module<br> _(after flashing with [sa8x8-fw](https://github.com/OpenRTX/sa8x8-fw/releases/tag/v1.3.0))_ :
https://github.com/nakhonthai/ESP32APRS_T-TWR/commit/f9d3917f029f8f08bb916787f2c70cae8d6cf4b1

Read this thread for more details:
https://github.com/nakhonthai/ESP32APRS_T-TWR/pull/17
